src/templates/votelog-list-template.js

```diff
@@ -23,10 +24,12 @@ export const query = graphql`
           }
           title
           description_th
+          passed
```

Comment:
Please don't revive `passed` and `total_voter`. Both are calculated live, so we won't query them.

See `calculateVoteLog()` in `src/utils/index.js` #149

Reply:
thanks, removed it.